lib/patternfly/_opensans-fonts.scss

```diff
@@ -0,0 +1,31 @@
+/*
+ * Keep in sync with https://github.com/cockpit-project/cockpit/tree/master/src/base1/_opensans-fonts.scss
```

Comment:
I'm a bit surprised -- I thought we'd use RedHat fonts for everything in Cockpit now. Why still OpenSans? I thought that was only a thing still for connecting to RHEL 7 version of Cockpit? (195, before the PF4 move)

Reply:
We do use RedHat fonts in everything AFAIK. tbh I just included it because of misunderstanding  - when I asked recently if we need to keep shipping the OpenSans fonts with patternfly.css in dist/base1

Anyway - I will remove it from here.